### ISSUE TITLE
History of associations

### ISSUE THREAD

**Issue by asaunier** [c2corg/v6_ui#193]
We need an interface to display the history of associations made/dropped with the current document.
For instance http://www.camptocamp.org/summits/latestassociations/id/37283

**Comment by asaunier** [comment]
Is this really an extra issue? The history of association is actually mentioned in the "périmètre fonctionnel" document.

**Comment by fjacon** [comment]
oui, mais je regarde ce qui serait négociable. cette fonction est-ce importante, bcp utilisée ?

**Comment by stef74** [comment]
@desnoes do you need something ?

**Comment by desnoes** [comment]
@stef74 : no i didn't do anything concerning this issue yet

**Comment by asaunier** [comment]
See also the API issue at https://github.com/c2corg/v6_api/issues/729
